Xamarin.Forms.Platform.Android/VisualElementRenderer.cs

```diff
@@ -8,6 +8,8 @@
 using Xamarin.Forms.Internals;
 using AView = Android.Views.View;
 using Xamarin.Forms.Platform.Android.FastRenderers;
+using Android.Runtime;
+using System.Linq;
```

Comment:
I wonder if we can do this without Linq. We do try to avoid it.

Reply:
Linq is avoided only on Android or anywhere?